components/filter/FilterGroup.tsx

```diff
@@ -75,6 +75,7 @@ const FilterOption = ({
     >
       <div className="inline-block h-3.5 w-6 text-sswRed">
         <Transition
+          as={"div"}
```

Comment:
to make consistent with others removed curly brackets
```suggestion
          as="div"
```